app/javascript/app/platform-authenticator.js

```diff
@@ -0,0 +1,13 @@
+function platformAuthenticator() {
+  const isEnabled = document.querySelector('[data-platform-authenticator-enabled]');
```

Comment:
I can see why the feature flag is here since this lets us include this in the JS bundle without having to worry about the feature flag at asset compile time. Still feels a bit odd since it trusts the client.